bluebrain/deployment/environments/applications_hpc.yaml

```diff
@@ -57,6 +52,7 @@ spack:
     - spatial-index
     - spykfunc
     - steps
```

Comment:
This steps now does not use `petsc`?

Reply:
`+petsc` variant is the default in `steps` recipe. So both modules use PETSc. Maybe you prefer something more explicit like this?

```
- steps ^petsc~complex
- steps@5 ^petsc+complex
```

Reply:
Ah, ok. So then the package preference for `+petsc` was totally superfluous. Thanks for clarifying! Good as is, then.